### PR TITLE
stable release files

### PR DESCRIPTION
**Pull request by SarahAldt**

### CODE REVIEW

**Comment by rvanharen** [comment]
@SarahAldt Maybe use ewatercycle@esciencecenter.nl as e-mail address instead of mine.

File: era5cli/__version__.py [hunk 1]
```diff
@@ -10,4 +10,4 @@
               'Niels Drost', 'Fakhereh Alidoost', 'Bouwe Andela',
               'Jerom Aerts', 'Berend Weel', 'Rolf Hut')
 __email__ = 'r.vanharen@esciencecenter.nl'
```
Comment (rvanharen) [inline era5cli/__version__.py]:
```suggestion
__email__ = 'ewatercycle@esciencecenter.nl'
```

**Comment by SarahAldt** [comment]
> Please also update the version number in the documentation, it is still referring to the previous version:
> 
> https://github.com/eWaterCycle/era5cli/blob/3716b27d2adf146661176e319010cbce6c06a67a/docs/source/conf.py#L28



> Please also update the version number in the documentation, it is still referring to the previous version:
> 
> https://github.com/eWaterCycle/era5cli/blob/3716b27d2adf146661176e319010cbce6c06a67a/docs/source/conf.py#L28

Thanks, it is done.

**Comment by rvanharen** [comment]
> > Please also update the version number in the documentation, it is still referring to the previous version:
> > https://github.com/eWaterCycle/era5cli/blob/3716b27d2adf146661176e319010cbce6c06a67a/docs/source/conf.py#L28
> 
> Thanks, it is done.

@SarahAldt did you push, I don't see the commit changing the version number in the documentation?

**Pull request merged**